libbeat/_meta/config/output-elasticsearch.reference.yml.tmpl

```diff
@@ -86,7 +86,7 @@ output.elasticsearch:
 
   # {{.BeatName}} expects Elasticsearch to be the same version or newer than the Beat.
   # Lift the version restriction by setting allow_older_versions to true.
-  #allow_older_versions: false
+  #allow_older_versions: true
```

Comment:
> # {{.BeatName}} expects Elasticsearch to be the same version or newer than the Beat.
  # Lift the version restriction by setting allow_older_versions to true.

We need to change this text as well since the version restricting is removed.

> # {{.BeatName}} expects Elasticsearch to be the same version or newer than the Beat.
  # Lift the version restriction by setting allow_older_versions to true.

We need to change this text. Maybe something like the following. I would say we just remove the parameter but that is a breaking change.

```go
# Prevents {{.BeatName}} from connecting to older Elasticsearch versions. 
```